common/reducers/wallet.ts

```diff
@@ -22,18 +25,42 @@ export interface State {
 
 export const INITIAL_STATE: State = {
   inst: null,
-  balance: null,
+  balance: { isPending: false, unit: null, amount: null },
```

Comment:
Does the snapshot need to be updated for this?

Reply:
The [test](https://github.com/MyEtherWallet/MyEtherWallet/pull/334/files#diff-ca3d0ee2c7f40670c0116b0b69a2ea4dR19) does, the snapshots don't.